V3-Upgrade-Guide.md

```diff
@@ -0,0 +1,35 @@
+# SAP Cloud SDK for JavaScript Version 3 Upgrade Guide
+
+The purpose of this document is to collect information on Cloud SDK version 2 to version 3 migration information.
+It should include information on all steps a user needs to take when updating the SDK version from 2 to 3.
+
+This document should be written in a style which addresses the consumer of the SDK.
+It will eventually end up in the SDK docs portal and release notes for version 3.
+
+Please add you items below when creating a change which will involve manual tasks for the user when performing the upgrade.
+Add sections to the document as you see fit.
+
+<!-- Everything below this line should be written in the style of enduser documentation. If you need to add hints for SDK developers, to that above. -->
+
+## How to upgrade SAP Cloud SDK for JavaScript to Version 3
+
+This document collects information on how to upgrade Cloud SDK version 2 to version 3.
+If you encounter issues with the documentation or while upgrading, please [open an issue in our GitHub repository](https://github.com/SAP/cloud-sdk-js/issues/new/choose).
+
+## Update your project dependencies
+
+Search for occurrences of `@sap-cloud-sdk/[some module]` in your `package.json` files.
+Replace the `2.x.y` version numbers with the [most recent `3.x.y` version number](https://github.com/SAP/cloud-sdk-js/releases).
```

Comment:
```suggestion
Replace the version numbers with `^3`.
```
[req] https://github.com/SAP/cloud-sdk/blob/main/STYLEGUIDE.md#do-not-use-formats-xy-or-xyz

Reply:
[pp] Can we supply a `sed` (and windows equivalent) command to replace this with a single command?